cmd/openshift-install/destroy.go

```diff
@@ -71,6 +75,12 @@ func runDestroyCmd(directory string) error {
 		return errors.Wrap(err, "failed to remove state file")
 	}
 
+	tfStateFilePath := filepath.Join(directory, terraform.StateFileName)
+	err = os.Remove(tfStateFilePath)
+	if err != nil && !os.IsNotExist(err) {
+		return errors.Wrap(err, "failed to remove Terraform state")
+	}
```

Comment:
You might be able to tighten this up a little:

```
if err = os.Remove(tfStateFilePath); !os.IsNotExist(err) {
		return errors.Wrap(err, "failed to remove Terraform state")
	}
```

Reply:
```
if err = os.Remove(tfStateFilePath); !os.IsNotExist(err) {
```

We don't want to error on `nil`, and [`IsNotExist(nil)` is `false`][1].  I like separating the call from the error-handling conditionals, but I can squash down to:

```
if err = os.Remove(tfStateFilePath); err != nil && !os.IsNotExist(err) {
```

if folks see that as a blocker ;).

[1]: https://play.golang.org/p/9pWS6DcJLDw

Reply:
> We don't want to error on nil

Oh of course. I left out the most important condition. Nvm!